src/renderers/shared/fiber/ReactFiberClassComponent.js

```diff
@@ -91,6 +96,98 @@ module.exports = function(scheduleUpdate : (fiber: Fiber) => void) {
     return true;
   }
 
+  function getName(workInProgress: Fiber, inst: any): string {
```

Comment:
I guess that module doesn't use `inst.constructor` as a fallback. Hm. Maybe it should? Don't know. I'll land this for now while we figure that out.